Makefile

```diff
@@ -155,6 +155,12 @@ dist_release:
 # additional sanity checks and file movement.
 build_dist_release:
 	make x-build-dist
+	# Reduce binary size by compressing binaries.
+	# Currently errors with `Couldn't find DIE referenced by DW_AT_abstract_origin`
+	# dwz ${CARGO_TARGET_DIR}/release/tikv-server
+	dwz ${CARGO_TARGET_DIR}/release/tikv-ctl
+	objcopy --compress-debug-sections=zlib ${CARGO_TARGET_DIR}/release/tikv-server
```

Comment:
I test on a centos localhost, It seems compress by `--compress-debug-sections=zlib`,  when I `gdb attach tikv-server` the backtrace lost source code line number info in compare to `--compress-debug-sections=zlib-gnu`, so why we choose `zlib` instead of `zlib-gnu`

Reply:
Whoa! Nice catch!